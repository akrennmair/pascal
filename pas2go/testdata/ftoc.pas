program ftoc;

var fgr, cgr : real;

begin
	write('Degree F: ');
	readln(fgr);
	cgr := (fgr-32.0)*5.0/9.0;
	writeln('Degree C: ', cgr:7:2)
end.
