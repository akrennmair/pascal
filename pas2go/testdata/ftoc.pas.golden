package main

import (
	"github.com/akrennmair/pascal/pas2go/system"
)

var _ = system.Write

// program ftoc
func main() {
	var (
		fgr float64
		cgr float64
	)
	_ = fgr
	_ = cgr

	system.Write("Degree F: ")
	system.Readln(&fgr)
	cgr = (fgr - 32.0e0) * 5.0e0 / 9.0e0
	system.Writeln("Degree C: ", cgr)
}
